content/_data/authors/lovakush.adoc

```diff
@@ -0,0 +1,7 @@
+---
+name: "Lovakush"
+github: lovakush
+linkedin: lova kush
```

Comment:
LinkedIn ID should not include a space.  I believe that https://www.linkedin.com/in/lova-kush-4a568b1b6/ is you.  If that is correct, then this is the change that is needed.
```suggestion
linkedin: lova-kush-4a568b1b6
```